static/js/app.ts

```diff
@@ -1,7 +1,7 @@
 // It's important that this file gets loaded first
-import './syntaxLang-en';
-import './syntaxLang-es';
-import './syntaxLang-nl';
+/*import './syntaxLang-en';*/
```

Comment:
We have git to track history. Just remove these lines, they're easy enough to get back if we change our mind.

Reply:
Done !